Makefile

```diff
@@ -84,6 +84,14 @@ test:
 	    $(GOTEST) ./...); \
 	done
 
+.PHONY: integrationtest
```

Comment:
What's the benefit of having a separate Make target (and build tag) for these?

Reply:
If the tests take longer, or require a GCP account, it is probably a good thing to separate them, IMO.

Reply:
My thinking is like what David said. Also if we blindly run the tests when credentials are available, contributors (or ourselves) could accidentally write a bunch of metrics to their project without realizing it